projects/openapi-utilities/src/diff/diff.ts

```diff
@@ -84,13 +90,13 @@ export function diff(before: any, after: any): ObjectDiff[] {
     // for objects, we just use the key
     if (Array.isArray(before.value) && Array.isArray(after.value)) {
       const allValues = [...before.value, ...after.value];
-      const arrayIdFn: (v: any, i: number) => string = allValues.every((v) =>
-        isParameterObject(v)
```

Comment:
this is not really correct. It works most of the time, but it's not specific to the parameter arrays we are actually trying to catch.